tests/conftest.py

```diff
@@ -1,54 +1,25 @@
 import pytest
```

Comment:
Make sure you add a test to lower debt ratio and increase debt ratio.
here you have an example:
https://github.com/Macarse/yhegic/blob/master/tests/development/test_debt_limit.py#L29

Reply:
Will do